src/index.js

```diff
@@ -290,21 +322,27 @@ function clear(element) {
 
 async function type(element, text, {allAtOnce = false, delay} = {}) {
   if (element.disabled) return
```

Comment:
I haven't duplicated this disabled check in the loop to check `document.activeElement.disabled`, because I don't think the focus can change to a disabled element.
But, if it feels safer to check I'm happy to update.

Reply:
No worries, updated now.